docs/dxp/7.2.x/en/platform/app-builder/app-builder.md

```diff
@@ -0,0 +1,54 @@
+# Low-Code Application Development with App Builder
+
+Writing applications for Liferay DXP has never been easier. Introduced with Liferay DXP 7.3, App Builder provides a low-code application development solution. There are four concepts at the root of App Builder apps:
+
+- [Custom Objects](#custom-objects)
+
+- [Form Views](#form-views)
+
+- [Table Views](#table-views)
+
+- [Deployment](#deployment)
+
+These concepts are explained below. To skip straight to building an application, see the next article on [Creating Applications with App Builder](./creating-applications-with-app-builder.md).
+
+## Custom Objects
+
+Custom objects are the foundation of your App Builder applications. The custom object is like the model in a [Service Builder application](placeholder). The custom object defines what data fields can be contained in the application. For example, a Guestbook object would contain a _Name_ field and a _Message_ field, at least. To create applications based on a custom object, you have three tasks: create at least one form view, create at least one table view, and deploy the application.
+
+The data for a custom object is defined at the same time as creating its form views.
+
+> In future versions of the App Builder, out-of-the-box objects with default fieldsets will be bundled with Liferay DXP to jump-start your app development. 
+
+## Form Views
+
+A Form View defines what fields of the custom object are to be filled out by the app's users. The Liferay Forms application's form builder tool is leveraged in this pursuit. A custom object can have multiple form views, but each application based on the custom object will only have one form view.
+
+Adding fields to a form view automatically adds them to the custom object. Fields that are part of the custom object via another form view can be re-used by dragging the field from a dedicated section for the form builder that holds the custom objects existing fields.
+
+## Table Views
+
+Table views display the custom object's collected data. Display all of the form view's fields, or a subset. Every app has one table view, though the custom object can have as many as needed. A drag-and-drop interface lets you create table views from the existing fields of the custom object.
```

Comment:
Another thing is that this text only highlights one of the Table View's value.  Another importance of the Table View that I didn't see written in here is that it's through the Table View that users will be able to manage the App's entries, including:
- Access the option to create new entries;
- Access the details view of an entry;
- Edit and remove an entry.

Reply:
Got it, I'll add this info. 

Question: will future versions of app builder allow an app to have multiple table views?

Reply:
Yes.

In a near future, we plan to have multiple table views in one same App in two situations:
1. Apps with multiple objects (we would need a Table View for each object);
2. Apps with Workflow Management (depending on the status of an entry in a Workflow, differents Table Views can be assigned in the same App).